app/src/main/java/de/tum/in/tumcampusapp/component/ui/cafeteria/widget/MensaRemoteViewFactory.java

```diff
@@ -45,32 +46,27 @@ public void onDestroy() {
 
     @Override
     public int getCount() {
-        if (mensaMenu == null) {
-            return 0;
-        }
-        return mensaMenu.size();
+        return mMenus.size();
     }
 
     @Override
     public RemoteViews getViewAt(int position) {
-
-        CafeteriaMenu currentItem = mensaMenu.get(position);
+        CafeteriaMenu currentItem = mMenus.get(position);
         if (currentItem == null) {
             return null;
         }
-        RemoteViews rv = new RemoteViews(applicationContext.getPackageName(), R.layout.mensa_widget_item);
+
+        RemoteViews rv = new RemoteViews(mApplicationContext.getPackageName(), R.layout.mensa_widget_item);
 
         String menuContent = COMPILE.matcher(currentItem.getName())
                                     .replaceAll("")
                                     .trim();
         rv.setTextViewText(R.id.menu_content, menuContent + " (" + currentItem.getTypeShort() + ")");
```

Comment:
uuhh, strings resources would probably be nice

Reply:
Fixed.